sonarts-core/src/rules/noMisleadingArrayReverseRule.ts

```diff
@@ -39,7 +40,10 @@ export class Rule extends tslint.Rules.TypedRule {
     typescriptOnly: false,
   };
 
-  public static MESSAGE = 'Move this array "reverse" operation to a separate statement.';
+  public static getMessage(methodName: string) : string
+  {
```

Comment:
@michalb-sonar we put curly brace on previous line. Anyway, just run `yarn format` before commit, it will do the job.